layouts/DocsPage/DocsPage.tsx

```diff
@@ -55,6 +63,43 @@ const DocsPage = ({
 
   const isOldVersion = available.indexOf(current) < available.indexOf(latest);
   const isBetaVersion = available.indexOf(current) > available.indexOf(latest);
+  let redirectFromBeta = false;
```

Comment:
I think that this code is repeating the code in `utils/findExistingPage`, can we replace it with this function too?